packages/core/lottie-react-native.podspec

```diff
@@ -23,7 +23,7 @@ Pod::Spec.new do |s|
   s.author                  = package["author"]
   s.homepage                = package["homepage"]
 
-  s.ios.deployment_target   = '12.4'
+  s.ios.deployment_target   = '13.0'
   s.osx.deployment_target   = '10.12'
   s.tvos.deployment_target  = '12.4'
```

Comment:
Please upgrade these two as well to match Lottie iOS:

  s.osx.deployment_target = '10.15'
  s.tvos.deployment_target = '13.0'